### ISSUE TITLE
where do clients enter passwords?

### ISSUE THREAD

**Issue by jasondaigo** [inolen/quakejs#73]
when i start the server with seta g_password where are clients suppose to enther the password? can they put it somewhere in the connect url?    
thanks in advance

**Comment by briancullinan** [comment]
Yes. Enter the standard url http://www.quakejs.com/play?connect%2034.107.80.109:27960&set%20g_password%20password123

**Comment by jasondaigo** [comment]
does this still work? i tried various combinations now and it always gives me invalid password somehow :-(
i can only connect without pass. i tried these settings in server.cfg:
`seta g_password "pass"`
`seta g_password pass`
`set g_password "pass"`
`set g_password pass`

maybe the url has to be different?

**Comment by briancullinan** [comment]
Try adding

g_needpass 1
